dagmc_stats.py

```diff
@@ -151,3 +151,46 @@ def get_surfaces_per_volume(my_core, entityset_ranges):
     for volumeset in entityset_ranges['Volumes']:
         s_p_v[volumeset] = my_core.get_child_meshsets(volumeset).size()
     return s_p_v
+
+
+def get_triangle_aspect_ratio(my_core, meshset):
+    """
+    Gets the triangle aspect ratio (according to the equation: (abc)/(8(s-a)(s-b)(s-c)), where s = .5(a+b+c).)
+    
+    inputs
+    ------
+    my_core : a MOAB Core instance
+    entityset_ranges : a dictionary with one entry for each entityset type, and the value is the range of entities that corrospond to 
+                        each type
+    meshset : a meshset containing a certain part of the mesh
+    
+    outputs
+    -------
+    t_a_r : (list) the triangle aspect ratios for all triangles in the meshset
+    """
+    
+    t_a_r = []
+
+    tris = my_core.get_entities_by_type(meshset, types.MBTRI)
+    
+    for triangle in tris:
+        side_lengths = []
+        s = 0
+        coord_list = []
+        
+        verts = list(my_core.get_adjacencies(triangle, 0))
+        
+        for vert in verts:
+            coords = my_core.get_coords(vert)
+            coord_list.append(coords)
+        
+        for side in range(3):    
+            side_lengths.append(np.linalg.norm(coord_list[side]-coord_list[side-2]))
```

Comment:
definitely more readable!!  Maybe a little too clever :smile: 

This probably deserves a comment like:
```
# triangle sides go between pairs of vertices with IDs: (0,1), (1,2), (2,0)
# if interpreted as indices into a list of length 3, these can be accomplished with (0,-2), (1,-1), (2,0), or (side, 2-side)
```

Reply:
Note that we don't need comments in most places because good variable names and data structure choices make the code more obvious to follow.  I think this is probably a case where it is a reasonable approach, but not obvious to follow.